php/meterpreter/ext_server_stdapi.php

```diff
@@ -368,7 +368,11 @@ function stdapi_fs_chdir($req, &$pkt) {
 function stdapi_fs_delete($req, &$pkt) {
     my_print("doing delete");
     $path_tlv = packet_get_tlv($req, TLV_TYPE_FILE_NAME);
-    $ret = @unlink(cononicalize_path($path_tlv['value']));
```

Comment:
cononicalize_path - is this what it is really called? or has delete always been broken in PHP?

Reply:
Oops. Good spot!

Reply:
Oh maybe not. It's a function we define ourselves further up. Typo though

Reply:
fixed anyhow :)